backends/openvino/tests/ops/base_openvino_op_test.py

```diff
@@ -0,0 +1,167 @@
+        compile_spec = [CompileSpec("device", self.device.encode())]
+        lowered_module: EdgeProgramManager = to_edge_transform_and_lower(
+            aten_dialect,
+            partitioner=[
+                OpenvinoPartitioner(compile_spec),
+            ],
+        )
+
+        # Apply backend-specific passes
+        exec_prog = lowered_module.to_executorch(
+            config=executorch.exir.ExecutorchBackendConfig()
+        )
+
+        # Check if the number of partitions created matches the expected number of partitions
+        self.assertEqual(
+            len(exec_prog.executorch_program.execution_plan[0].delegates),
+            expected_partitions,
+        )
+        # Check if the individual partitions are assigned to Openvino backend
+        for i in range(expected_partitions):
+            self.assertEqual(
+                exec_prog.executorch_program.execution_plan[0].delegates[i].id,
+                OpenvinoBackend.__name__,
+            )
+
+        # Execute the model and compare the outputs with the reference outputs
+        if assert_output_equal:
+            with tempfile.TemporaryDirectory() as tmp_dir:
+                input_list = ""
+                for idx, _ in enumerate(sample_inputs):
+                    input_name = f"input_0_{idx}.raw"
+                    input_list += input_name + " "
+                input_list = input_list.strip() + "\n"
+
+                output_dir = f"{tmp_dir}/outputs"
+
+                # Execute the module in eager mode to calculate the reference outputs
+                ref_output = module(*sample_inputs)
+                if isinstance(ref_output, torch.Tensor):
+                    ref_output = [
+                        ref_output,
+                    ]
+
+                # Serialize the executorch model and save into a temporary file
+                pte_fname = f"{tmp_dir}/openvino_executorch_test.pte"
+                with open(pte_fname, "wb") as file:
+                    exec_prog.write_to_file(file)
+
+                # Save inputs into a temporary file
+                self.generate_inputs(
+                    tmp_dir, "input_list.txt", [sample_inputs], input_list
+                )
+                self.make_output_dir(output_dir)
+
+                # Start a subprocess to execute model with openvino_executor_runner
+                cmd = [
+                    f"{self.build_folder}/examples/openvino/openvino_executor_runner",
+                    "--model_path",
+                    pte_fname,
+                    "--input_list_path",
+                    f"{tmp_dir}/input_list.txt",
+                    "--output_folder_path",
+                    output_dir,
+                ]
+
+                env = dict(os.environ)
+                proc = subprocess.run(
+                    cmd,
+                    stdout=subprocess.PIPE,
+                    stderr=subprocess.STDOUT,
+                    env=env,
```

Comment:
I also dont quite like doing this unit tests. DOesnt sound right to me.

Reply:
Is this done executor runner sanity check?

Reply:
We have changed tests to use python bindings. We do not need to use subprocess anymore, so we removed this.